bindings/python/iota_sdk/types/address.py

```diff
@@ -91,3 +76,34 @@ class AddressWithUnspentOutputs():
     key_index: int
     internal: bool
     output_ids: bool
+
+
+AddressUnion: TypeAlias = Union[Ed25519Address, AccountAddress, NFTAddress]
```

Comment:
Can we just name this `Address`? And same for others?

Reply:
Would be conflicting with `class Address():`

Reply:
Ah, bummer